beacon-chain/db/iface/interface.go

```diff
@@ -99,6 +99,11 @@ type HeadAccessDatabase interface {
 	// Block related methods.
 	HeadBlock(ctx context.Context) (*eth.SignedBeaconBlock, error)
 	SaveHeadBlockRoot(ctx context.Context, blockRoot [32]byte) error
+
+	// Genesis operations.
+	LoadGenesisFromFile(ctx context.Context, filePath string) error
```

Comment:
use io.Reader here or ReadCloser, makes it more extensible. Imagine in the future someone wants to read the genesis state from some other source that isn't a file